server/migrations/20210722181933_unpadded_ext_ids.js

```diff
@@ -0,0 +1,17 @@
+exports.up = async function (knex) {
+  const zeroPaddedRows = await knex("external_ids")
+    .select("provider_location_id", "system", "value")
+    .where("system", "cvs")
+    .where("value", "~", "^0\\d+$");
```

Comment:
I was thinking this needed to be double-escaped, but I guess not!

Side note: `String.raw` is nice for this kind of stuff:

```suggestion
    .where("value", "~", String.raw`^0\d+$`);
```